src/Ast/Type/ArrayShapeUnsealedTypeNode.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\PhpDocParser\Ast\Type;
+
+use PHPStan\PhpDocParser\Ast\NodeAttributes;
+use function sprintf;
+
+class ArrayShapeUnsealedTypeNode implements TypeNode
```

Comment:
This should be just a Node, not TypeNode. It doesn't make sense standalone.

Reply:
Makes sense. `ArrayShapeItemNode` shouldn't have been one either then I guess, oh well.
Done.